src/main/core/worker.h

```diff
@@ -76,8 +76,8 @@ const ConfigOptions* worker_getConfig();
 gboolean worker_scheduleTask(Task* task, Host* host, SimulationTime nanoDelay);
 void worker_sendPacket(Host* src, Packet* packet);
 bool worker_isAlive(void);
-// Maximum time that the current thread may run ahead to.
-EmulatedTime worker_maxThreadTime(Host* host);
+// Maximum time that the current event may run ahead to.
+EmulatedTime worker_maxEventRunaheadTime(Host* host);
```

Comment:
There is also the `worker_updateMinRunahead()` function, so maybe one of these functions should change names, or they should have documentation explaining that they refer to different runaheads.

Reply:
Changed `worker_updateMinRunahead` to `worker_updateMinHostRunahead`